src/main/kotlin/commands/CapeCommand.kt

```diff
@@ -0,0 +1,463 @@
+                                message.error(changeError(capeUUID, it))
+                                return@doesLater
+                            }
+
+                            val oldCapeColor = cape.color
+                            cape.color = CapeColor(colorPrimary.toLowerCase(), colorBorder.toLowerCase())
+
+                            val oldColors = oldCapeColor.toEmoji()
+                            val newColors = cape.color.toEmoji()
+
+                            message.channel.send {
+                                embed {
+                                    description = "Successfully changed the colors of Cape `${cape.capeUUID}`!"
+                                    field(
+                                        "Old Colors",
+                                        oldColors,
+                                        true
+                                    )
+                                    field(
+                                        "New Colors",
+                                        newColors,
+                                        true
+                                    )
+                                    color = Colors.success
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        literal("save") {
+            doesLaterIfHas(PermissionTypes.AUTHORIZE_CAPES) {
+                save()
+                commit()
+                message.success("Saved!")
+            }
+        }
+
+        literal("load") {
+            doesLaterIfHas(PermissionTypes.AUTHORIZE_CAPES) {
+                load()
+                message.success("Loaded!")
+            }
+        }
+
+        load()
+    }
+
+
+    private fun load() {
+        if (!File(capesFile).exists()) return
+        try {
+            Files.newBufferedReader(Paths.get(capesFile)).use { bufferedReader ->
+                val cacheList = Gson().fromJson<List<CapeUser>>(bufferedReader, object : TypeToken<List<CapeUser>>() {}.type)
+                capeUserMap.clear()
+                capeUserMap.putAll(cacheList.associateBy { it.id })
+            }
+        } catch (e: Exception) {
+            log("Error loading capes!")
+            e.printStackTrace()
+        }
+    }
+
+    fun save() {
+        val capeUsers = capeUserMap.values.toList()
+        if (!File(capesFile).exists()) File(capesFile).createNewFile()
+
+        Files.newBufferedWriter(Paths.get(capesFile)).use {
+            it.write(gson.toJson(capeUsers, object : TypeToken<List<CapeUser>>() {}.type))
+        }
+    }
+
+    suspend fun commit() { // TODO: hardcoded and a hack. I'm embarrassed to push this, but this is waiting for me to add plugin support
+        readConfigSafe<UserConfig>(ConfigType.USER, false)?.primaryServerId?.let {
+            if (it != 573954110454366214) return
+        } ?: return
```

Comment:
You can just replace this with a simple if check instead because `null != 573954110454366214` will always be true.

```suggestion
        if (readConfigSafe<UserConfig>(ConfigType.USER, false)?.primaryServerId != 573954110454366214) return
```